scope_shared/scope/testing/fake_data/fixtures_fake_patient.py

```diff
@@ -400,174 +400,174 @@ def data_fake_identity_factory() -> dict:
 #     return fake_values_inventory
 
 
-def data_fake_safety_plan_factory() -> dict:
-    fake_safety_plan = {
-        "_type": "safetyPlan",
-        "_rev": 1,
-        "assigned": True,
-        "assignedDate": "some date",
-        "reasonsForLiving": "To stare at Mt. Rainier.",
-        "supporters": [
-            {
-                "contactType": "Person",
-                "name": "Name",
-                "address": "Address",
-                "phoneNumber": "Number",
-            }
-        ],
-    }
+# def data_fake_safety_plan_factory() -> dict:
```

Comment:
I believe this entire file is not in use, is not defined as we bring more of our "real" data online.

I think you're keeping it and commenting it as a reference while you work through things.  Fine w/ me.